lib/slack/real_time/concurrency/celluloid.rb

```diff
@@ -38,6 +38,8 @@ def run_loop
           rescue EOFError => e
             logger.debug("#{self.class}##{__method__}") { e }
             driver.emit(:close, WebSocket::Driver::CloseEvent.new(1001, 'server closed connection')) unless @closing
+          ensure
+            terminate
```

Comment:
I don't think so because in this case this class is the actor and there's literally nothing terminating it ever after even a "successful" run loop. Note that all run loops end with an EOF or some error, there's not even such a thing as clean termination. I could be wrong :)